src/App.tsx

```diff
@@ -1,13 +1,46 @@
 import React from 'react';
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import TabsNavigation from './components/TabsNavigation/TabsNavigation';
 
 function App() {
-  return (
-    <div className="App">
-      <!-- your component goes here to test the rendering! -->
-    </div>
-  );
+    return (
+        <div className="App">
+            {/* <!-- your component goes here to test the rendering! --> */}
+            <TabsNavigation>
+                {[
```

Comment:
Could we make this read a little easier by using a child component instead here?
```suggestion
                <Tab title="Prospects" badge="99"/>
```

Reply:
@kkyeboah Sure! I finally made a new PR enabling these changes.

It was a bit trickier than it may look, since the "onChange" props of the "Tabs" Material-UI component was not working properly. At the end I found a way to fix it, and it was pretty simple.

In any case I will leave here the link where I found the answer, so if we find a similar problem we can quickly reach the solution:

https://github.com/mui-org/material-ui/issues/13893